SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -289,12 +289,12 @@ void ActorSystem::Advance()
 
 	for (const FEntitySubView& SubView : SubViews)
 	{
-		ProcessRemoves(SubView);
+		ProcessUpdates(SubView);
```

Comment:
TearOff is applied during ProcessUpdates; should we receive both TearOff and EntityDelta::REMOVE, we can remove entity first, leading to it being destroyed.

Reply:
Is that not correct behaviour? If something was torn off and left your view at the same time it would seem it's correct to remove it from the view although I guess with tear off this is a race. Is it ever possible that we don't want to process updates because something was removed? As far as I can tell no, in which case maybe we can pick either behaviour.